src/app/containers/Ad/Canonical/index.test.jsx

```diff
@@ -35,34 +45,45 @@ describe('CanonicalAds Ads', () => {
 
 jest.mock('#lib/utilities/isLive', () => jest.fn());
 
-describe('getBootsrapSrc', () => {
+describe('getBootstrapSrc', () => {
+  beforeEach(() => {
+    process.env.SIMORGH_ADS_SCRIPT_TEST = mockTestScript;
```

Comment:
Is there a way to just use 1 `beforeEach` for both of these describe blocks?

Reply:
Yup done!